generators/python-v2/ast/src/Reference.ts

```diff
@@ -67,7 +68,11 @@ export class Reference extends AstNode {
         }
     }
 
-    public getFullyQualifiedModulePath(): string {
+    public getFullyQualifiedPath(): string {
         return this.modulePath.join(".");
     }
+
+    public getCompletePath(): string {
+        return `${this.getFullyQualifiedPath()}.${this.name}`;
+    }
```

Comment:
nit: I would use `getFullyQualifiedModulePath` and `getFullyQualifiedPath` (since omitting the `Module` component implies its for the reference), but feel free to ignore if you disagree. Not a big deal either way, and it's easy to change later (if ever).